src/sentry/snuba/metrics/extraction.py

```diff
@@ -123,6 +123,8 @@
     "failure_count": "on_demand_failure_count",
     "failure_rate": "on_demand_failure_rate",
     "apdex": "on_demand_apdex",
+    "epm": "on_demand_epm",
+    "eps": "on_demand_eps",
```

Comment:
Should you also add mappings for `tpm` and `tps` here, or is that converted somewhere upstream of the extraction module?

Reply:
`tpm` and `tps` cannot be used in widgets.
It's a concept within Performance transaction summaries.